Magento_Theme/templates/custom_footer.phtml

```diff
@@ -0,0 +1,6 @@
+<?php $block = $this->getLayout()->getBlock('custom.footer'); ?>
```

Comment:
you already have this class in the `$block` variable. You don't have to do this. To help IDE to find the class, use this instead
```php
<?php

/**
 * @var Magento\Framework\View\Element\Template $block
 */
?>
```